### DISCUSSION TITLE
Type on a pricing page

### DISCUSSION THREAD

**Discussion by AlekSi** [ubicloud/ubicloud#1183] [General]
https://www.ubicloud.com/docs/github-actions-integration/price-performance

![](https://assets-global.website-files.com/64f9d9b4e737e7b37d4e39a4/65a71ce57a10a849b145d303_Ubicloud-runner-prices-x64png.png)

Assuming that `ubuntu-latest` and `ubicloud-standard-2` are aliases, the first row misses one `0` in the price.

**Comment by byucesoy** [comment]
Hi @AlekSi.

`ubuntu-latest` and `ubicloud-standard-2` are not alias of each other. `ubuntu-latest` is the option for using Github's own runners and `ubicloud-standard-2` is the option for using Ubicloud's runners. Ubicloud runner is 10x more cost effective than comparable Github runner. So it is not a typo, missing 0 

I guess the source of the confusion is that all entries are in single table and one would expect all of them are different options of Ubicloud runners, whereas the first row is an option of Github runners and the rest are option of Ubicloud runners. We put both to same table for easy comparison, but probably lack of visual clue made that less obvious. Maybe a horizontal line between first and second row would emphasize that. What do you think about it?

**Reply by AlekSi** [reply]
I wonder if you could add another column showing the price of the equivalent GitHub instance for each of your instances.
